src/fights.ts

```diff
@@ -2247,8 +2248,21 @@ function yachtzee(): void {
 
       if (!equippedOutfit || !success()) return;
 
-      setChoice(918, 2);
+      const lastUMDDate = property.getString("umdLastObtained");
+      const today = todayToString().slice(0,4) + "-" + todayToString().slice(4,6) + "-" + todayToString().slice(6);
```

Comment:
there is almost certainly a better way to do this

Reply:
I'm not familiar with JS so I don't really know what the preferred coding style is / relevant standard libraries are to do date conversions. Could you advise on a better implementation?